kinto_attachment/tests/config/seafile.ini

```diff
@@ -0,0 +1,14 @@
+[app:main]
+use = egg:kinto
+kinto.userid_hmac_secret = some-secret-string
+
+kinto.includes = kinto.plugins.default_bucket kinto_attachment
+
+kinto.attachment.base_url = https://seafile.net
+
+kinto.attachment.folder = {bucket_id}/{collection_id}
+
+kinto.attachment.seafile.servname = https://seafile.net
+kinto.attachment.seafile.username = seafile
```

Comment:
I would rather go with the bearer token an see if we have any problem. (if it is API tokens we shouldn't because you should be able to have a token per app using your service.)